Makefile

```diff
@@ -17,19 +17,23 @@ clean:
 
 ontologies: $(ONTFILES)
 
-ontologies/mondo-issue-%.owl:
-	mkdir -p github && mkdir -p github/mondo-issue-$* && rm -rf github/mondo-issue-$*/*
-	cd github/mondo-issue-$* && git clone --depth 1 https://github.com/monarch-initiative/mondo.git -b issue-$* 
-	$(ROBOT) merge -i github/mondo-issue-$*/mondo/src/ontology/mondo-edit.obo --catalog github/mondo-issue-$*/mondo/src/ontology/catalog-v001.xml remove --select ontology reason --reasoner ELK -o $@.tmp.owl && mv $@.tmp.owl $@
+ontologies/mondo-branch-%.owl:
```

Comment:
curious, why did you change issue to branch? issue was a bit more clear, since branch is ambiguous between github and the ontology branches themselves

Reply:
Because I wanted to accommodate for arbitrary branch names, not just those starting with "issue". Previously I made the assumption that a branch starts with the string "issue", which is mostly, but not always, true.